Sources/Build/LLBuildManifestBuilder.swift

```diff
@@ -626,7 +626,8 @@ extension LLBuildManifestBuilder {
             tempsPath: target.tempsPath,
             objects: try target.objects,
             otherArguments: try target.compileArguments(),
-            sources: target.sources,
+            sourceCount: target.sources.count,
+            fileListPath: try target.writeFileList(target.sources),
```

Comment:
I would prefer we make the "writeFileList" functional accessible to this module than change the signature like this

Reply:
I'm not sure I understand what you are suggesting.  Do you want to pass `target` down and invoke `writeFileList` in the callee?  I will note that we do something similar a few lines below that I didn't change.

Reply:
was trying to eliminate having to pass "sourceCount". is it very useful? @neonichu ?

Reply:
Oh, I see!  Hmm, we would need to pass `target` down.  I do agree with the sentiment though - I did first try to not pass `sourceCount` down.  That is used entirely for display purposes though, not building.

Reply:
Don't think it is super useful, would be fine to remove it. I only put it there as I was trying to match the existing swift-compiler-tool behavior, but I don't think we have to do that going forward.